lib/html_to_markdown.rb

```diff
@@ -166,17 +166,18 @@ def visit_a(node)
 
   def visit_img(node)
     return if node["src"].blank?
+    title_or_alt = node["alt"].presence || node["title"].presence
+    title_or_alt = title_or_alt&.gsub(/\n/, " ")&.squeeze(" ")
```

Comment:
is this relevant in the first two `if` branches? (i.e. any reason it was moved from the last `if` branch?)

Reply:
I realised overnight I stupidly didn't fix this for the other branches even though this is what I set out to do :laughing: Fixing now...